staging/src/k8s.io/client-go/examples/create-update-delete-deployment/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	"os"
 	"path/filepath"
 
+	appsv1 "k8s.io/api/apps/v1"
 	appsv1beta1 "k8s.io/api/apps/v1beta1"
```

Comment:
It's not perfectly clear but going off of how `kubectl` [does rollbacks](https://github.com/kubernetes/kubernetes/blob/a73c96d7b28e8c04a5bd5f68af04eecea3ec14ff/pkg/kubectl/rollback.go) I think we may want to use `extensions/v1beta1` instead.

Reply:
@jekohk Okay got it, will update it.

Reply:
ext/v1beta1 is deprecated in favor of apps/v1 and others. If possible, we should not have example with the former anymore.

What do we need here in the example to get rid of appsv1beta1?

Reply:
@sttts as far as I can tell there is no deployment_expansion is apps/v1, meaning there is no type or func to perform the rollback there. Am I missing something?

Reply:
Then we should add it probably. I don't like to promote v1beta1 functionally here in the example.

@tnozicka can you comment on the missing rollback expansion in v1?